classes/order/OrderReturn.php

```diff
@@ -74,20 +81,17 @@ public function addReturnDetail($order_detail_list, $product_qty_list, $customiz
                 }
             }
         }
-        /* Customized product return */
-        if ($customization_ids) {
-            foreach ($customization_ids as $order_detail_id => $customizations) {
-                foreach ($customizations as $customization_id) {
-                    if ($quantity = (int) $customization_qty_input[(int) $customization_id]) {
-                        Db::getInstance()->insert('order_return_detail', ['id_order_return' => (int) $this->id, 'id_order_detail' => (int) $order_detail_id, 'product_quantity' => $quantity, 'id_customization' => (int) $customization_id]);
-                    }
-                }
-            }
-        }
     }
 
-    public function checkEnoughProduct($order_detail_list, $product_qty_list, $customization_ids, $customization_qty_input)
+    public function checkEnoughProduct($order_detail_list, $product_qty_list, $customization_ids = null, $customization_qty_input = null)
     {
+        if ($customization_ids !== null || $customization_qty_input !== null) {
+            @trigger_error(
+                'Passing customization infos is deprecated since version 8.0.0. The customization is already included in the order details.',
```

Comment:
```suggestion
                'Passing customization infos is deprecated since version 9.0.0. The customization is already included in the order details.',
```